Organic/AmpAdsInjector.php

```diff
@@ -3,8 +3,22 @@
 namespace Organic;
```

Comment:
These changes just reflect the current master branch.

Reply:
Don't include changes from the master branch in your commits please. We'll have problems with merging them.
Use rebase instead. I've rebased the `feature/affiliate-link` branch, please update yours as well. Thank you